released/packages/coq-formalv-check_range/coq-formalv-check_range.1.3.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Tactics to automatically prove Boolean goals involving Uint63/Sint63 variables" # One-line description
+description: """
+FV Check Range provides tactics to automatically prove Boolean goals
+involving 1, 2 or 3 Uint63.int/Sint63.int bounded variables through
+brute-force computation.
+""" # Longer description, can span several lines
+
+homepage: "https://gitlab.com/formalv/formalv"
+dev-repo: "git+https://gitlab.com/formalv/formalv"
+bug-reports: "https://gitlab.com/formalv/formalv/-/issues"
+doc: "https://formalv.gitlab.io/formalv/coqdoc/toc.html"
+maintainer: "Mireia González Bedmar <mireia.gbedmar@formalv.com>"
+
+tags: [
+  "keyword:automation"
+  "keyword:primitive integers"
+  "logpath:formalv"
```

Comment:
```suggestion
  "logpath:formalv.check_range"
```
?

Reply:
This is a developper choice, even if everything in the package lies within `formalv.check_range`, they may want to recommend users to do `From formalv Require ...` and not `From formalv.check_range Require ...`, for instance to ease potential move of files between packages. So, indeed up to @quimFIB

Reply:
@proux01 I agree with Guillaume here, I think the ideal is that `logpath` in the package precisely reflects the prefix that applies (and thus where `.vo` files end up). Then this information can potentially be used to detect installation clashes and the like.